ProjectTemplates/AspNetCoreMicrosoftIdentityWebProjectTemplates.csproj

```diff
@@ -36,4 +36,11 @@
     <Content Include="templates\**\*" Exclude="templates\**\bin\**;templates\**\obj\**" />
     <Compile Remove="**\*" />
   </ItemGroup>
+
```

Comment:
I don't understand these 4 lines? 
I think it's a tooling (Visual Studio) issue.